src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -141,7 +141,7 @@ def __nonzero__(self):
         type, value, tb = sys.exc_info()
         if type is not None and issubclass(type, py_compile.PyCompileError):
             print "Problem with", repr(value)
-            raise type, value, tb
+            raise type(value).with_traceback(tb)
         return 1
```

Comment:
Is this 2 and 3 compatible?

Also, I have code for annotating exceptions in sasmodels.exceptions that maybe useful here.